lib/pure/distros.nim

```diff
@@ -173,7 +177,8 @@ proc detectOsImpl(d: Distribution): bool =
       case d
       of Distribution.Gentoo:
         result = ("-" & $d & " ") in uname()
-      of Distribution.Elementary, Distribution.Ubuntu, Distribution.Debian, Distribution.Fedora,
+      of Distribution.Elementary, Distribution.Ubuntu, Distribution.Debian,
+          Distribution.Fedora,
         Distribution.OpenMandriva, Distribution.CentOS, Distribution.Alpine,
         Distribution.Mageia, Distribution.Zorin:
```

Comment:
That's some inconsistent formatting...

Reply:
This is what `nimpretty lib/pure/distros.nim` produced. I'll format it back, but seems like a bug in that.